src/Data/List/Relation/Unary/Any/Properties.agda

```diff
@@ -97,6 +97,11 @@ Any-cong {P = P} {Q = Q} {xs = xs} {ys} P↔Q xs≈ys =
 ------------------------------------------------------------------------
 -- Any.map
 
+map-ext : (f g : P ⋐ P) → (∀ {x} (p : P x) → f p ≡ g p) →
```

Comment:
I don't quite understand why `f g` can't be from `P` to `Q`?

Reply:
Indeed: see above. I don't think we need this lemma at all